stack-graphs/src/stitching.rs

```diff
@@ -283,6 +283,22 @@ impl Database {
     pub fn iter_partial_paths(&self) -> impl Iterator<Item = Handle<PartialPath>> {
         self.partial_paths.iter_handles()
     }
+
+    pub fn ensure_both_directions(&mut self, partials: &mut PartialPaths) {
+        for path in self.partial_paths.iter_handles() {
+            self.partial_paths
+                .get_mut(path)
+                .ensure_both_directions(partials);
+        }
+    }
+
+    pub fn ensure_forwards(&mut self, partials: &mut PartialPaths) {
+        for path in self.partial_paths.iter_handles() {
+            self.partial_paths
+                .get_mut(path)
+                .ensure_both_directions(partials);
```

Comment:
Is this supposed to have the same implementation as `ensure_both_directions`, or should it call `ensure_forwards`?

Reply:
It should call `ensure_forwards`. (It would've worked like this, but with possibly doing too much work.)